app/views/forever_style_guide/sections/components/navigation/_nav_help.erb

```diff
@@ -0,0 +1,10 @@
+<li class='dropdown'>
+  <%= link_to nil, class: 'dropdown-toggle dropdown-toggle-help', :data => { :toggle => 'dropdown' } do %>
+    Help
+    <i class='fa fa-chevron-down'></i>
+  <% end %>
+
+  <div class='dropdown-menu dropdown-menu-nav dropdown-menu-deals color_block-gray-700'>
```

Comment:
this is misleading that the dropdown-menu-deals class is applied to the help link